addons/medical_damage/initSettings.sqf

```diff
@@ -24,3 +24,12 @@
     [0, 25, 1, 2],
     true
 ] call CBA_settings_fnc_init;
+
+[
+    QEGVAR(medical,painUnconsciousChance),
+    "SLIDER",
+    [LSTRING(PainUnconsciousChance_DisplayName), LSTRING(PainUnconsciousChance_Description)],
+    ELSTRING(medical,Category),
+    [0, 1, 0.1, 2, true],
+    true
+] call CBA_settings_fnc_init;
```

Comment:
`CBA_settings_fnc_init` should be replaced with `CBA_fnc_addSetting` here and above

Reply:
I'd say do that in separate PR ACE-wide. All medical still uses the old function.